b9lab-content/2-what-is-cosmos/03-cosmos-ecosystem.md

```diff
@@ -0,0 +1,177 @@
+
+<HighlightBox type="tip">
+
+If you want to explore more on application-specific blockchains, take a look at the section on **[A Blockchain App Architecture]()**.
+--> Include link to mod. 2, section 1
+
+</HighlightBox>
+
+Focusing on customizability and interoperability leads to establishing a stable universe determined by the same rules that apply to the whole ecosystem equally.
+
+--> Include hotspot image of ecosystem
+
+## The Cosmos SDK - Modularity and Customizing
+
+The Cosmos SDK can be understood as a generalized framework to build secure blockchain applications on the Tendermint BFT in Golang. It is a modular framework for application-specific blockchains. The design bases itself on **two major principles**: modularity and capability-based security.
+
+The SDK was envisioned to be an npm-like framework for secure applications on top of Tendermint. It has become an advanced framework for custom application-specific blockchains.
+
+The aim of the SDK is to create an ecosystem of components that not only help spin up an application-specific blockchain but that also are customizable. With the SDK, developers no longer have to code every part of the blockchain. The Cosmos SDK includes tools to build command-line interfaces (CLIs), REST servers, and utility libraries.
+
+The ready-built modules of the Cosmos SDK can be easily used - It is as simple as importing modules into the application. Additionally, developers can create their own modules to introduce specific functionalities. It is expected that, with the growth of the ecosystem, the number of modules will grow, thereby facilitating the development of more complex applications.
+
+*Building on modular components, many of those you did not write yourself? Although it sounds like an increased potential for attacks, and faulty or malicious nodes operating undetected? One does not have to worry.*
+
+The Cosmos SDK is built on a foundation of the [object-capability model](https://docs.cosmos.network/master/core/ocap.html). It not only favors modularity, it also encapsulates code implementation. Thus, capabilities constrain security boundaries between modules.
+
+The default consensus mechanism available to build with the SDK is [Tendermint Core](https://docs.tendermint.com/master/).
+
+## The Inter-Blockchain Communication Protocol
+
+The [Inter-Blockchain Communication (IBC) protocol](https://ibcprotocol.org/) is the basis for **interoperability** in Cosmos. It leverages Tendermint's instant finality to allow for the transfer of value (i.e. tokens) and data communication between heterogenous chains. Blockchains with different applications and architecture specifications become interoperable whether or not they share a validator set.
```

Comment:
Note it also work with non-Tendermint chains, even without instant finality. The ICS 02 Client interface is generic over consensus. ICS 07 is the Tendermint implementation for this, but there are others